src/neural/dx/network_dx.cc

```diff
@@ -587,6 +587,43 @@ DxNetwork::DxNetwork(const WeightsFile& file, const OptionsDict& options)
     network_.emplace_back(std::move(FCVal2));
   }
 
+  // Moves left head
+  moves_left_ = file.format().network_format().moves_left() ==
+                pblczero::NetworkFormat::MOVES_LEFT_V1;
+  if (moves_left_) {
+    // 1x1 convolution, moves_left biases output filters
+    auto convMov = std::make_unique<ConvLayer>(
+        fp16_, nullptr, nullptr, &dx_context_, resi_last,
```

Comment:
Note that the value head has getLastLayer() here instead of resi_last - I think that is a bug that luckily has no negative outcomes...

Reply:
You are right - better change that too (although not affecting anything right now).

Reply:
Done.